compiled_starters/redis-starter-java/src/Main.java

```diff
@@ -0,0 +1,29 @@
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+public class Main {
+  public static void main(String[] args){
+    System.out.println("Your code goes here!");
+
+    //  Uncomment this block to pass the first stage
+    //    ServerSocket serverSocket = null;
+    //    Socket socket = null;
```

Comment:
Would you be okay with renaming `socket` to `clientSocket`? I think that'll make the intention a bit more obvious 🙂